eng/common/scripts/SemVer.ps1

```diff
@@ -22,6 +22,7 @@ class AzureEngSemanticVersion {
   [string] $RawVersion
   [bool] $IsSemVerFormat
   [string] $DefaultPrereleaseLabel
+  [string] $ParseLanguage = $Language
```

Comment:
This should probably be a static variable so that we can control outside the scope of an individual object.